packages/bot-web-ui/src/components/chart/v2.ts

```diff
@@ -8,7 +8,7 @@ const init = () => {
         // TODO: Proper fix for types in smartcharts
         // eslint-disable-next-line @typescript-eslint/ban-ts-comment
         // @ts-ignore
-        return import(/* webpackChunkName: "smart_chart_beta" */ '@deriv/deriv-charts-beta');
+        return import(/* webpackChunkName: "smart_chart_beta" */ '@deriv/deriv-charts');
```

Comment:
```suggestion
        return import(/* webpackChunkName: "smart_chart_bala" */ '@deriv/deriv-charts');
```
nit: probably remove the word beta from the chunk name